README.md

```diff
@@ -0,0 +1,14 @@
+## Purpose:
+To send notification messages to various discord channels, reminding people that there is an event today.
+
+## Tools Used
+- Node
+
+## Setting Up your Environment
+- Install Node 16.
```

Comment:
Link? I'd probably make this more generic in that to just install some latest stable node.

Reply:
Great point, done!